woocommerce/woocommerce
# Test for adding a complex order

### All Submissions:


### Changes proposed in this Pull Request:

Closes #30674.

### How to test the changes in this Pull Request:

1. Clone this branch and launch your local e2e environment.
2. Run the orders tests: `npm test -- --group=orders`
3. You should see `order-complex.test.js` pass:
```
PASS  tests/orders/order-complex.test.js (5.076s)
  Orders API test
    ✓ can add complex order (675ms)
```

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### FOR PR REVIEWER ONLY:
